inst/extdata/PA2024CH_de_exec_summary/template.Rmd

```diff
@@ -540,9 +540,9 @@ Die folgende Grafik zeigt die Anforderungen in Bezug auf den Gesamtscore zur Kli
 
 \newpage
 
-\section {Anhang (II): zusätzliche Erläuterungen zum Diagramm - Umfrage zu Klimamassnahmen}
+\section {Anhang (II): Zusätzliche Erläuterungen zum Diagramm - Umfrage zu Klimamaßnahmen}
```

Comment:
These still show up as 'ss'. Maybe indeed you need to follow the advice from @jdhoffa and use unicode characters.